### DISCUSSION TITLE
Current Layout Changes

### DISCUSSION THREAD

**Discussion by LWFlouisa** [mastodon/mastodon#30419] [Q&A]
I figured out how to make my profile public again, but currently I get a notification that says my profile isn't locked, and I can't seem to figure out how to set my default profile setting to public.

Has anyone else encountered this issue? It's not just one instance here, it's on three different ones I'm on.

**Comment by vmstan** [comment]
This warning is probably being triggered by your post settings.

![CleanShot 2024-05-28 at 08 32 29@2x](https://github.com/mastodon/mastodon/assets/3002053/8ab22f87-b31f-4b99-a172-e9735c6c502d)
